src/dialogs/alwaysonbotDialogs/Common/feedBackStep.ts

```diff
@@ -0,0 +1,51 @@
+import {
+    Choice, ChoiceFactory, ChoicePrompt, ComponentDialog, DialogTurnResult, PromptValidatorContext, TextPrompt,
+    WaterfallDialog,
+    WaterfallStepContext
+} from "botbuilder-dialogs";
+
+import i18n from "../../locales/i18nconfig";
+
+const TEXT_PROMPT = "TEXT_PROMPT";
+const CHOICE_PROMPT = "CHOICE_PROMPT";
+export const FEED_BACK_STEP = "FEED_BACK_STEP";
+const FEEDBACK_STEP_WATERFALL_STEP = "FEEDBACK_STEP_WATERFALL_STEP";
```

Comment:
line 11,12, the feedback format is not consistent, better change both to FEED_BACK or FEEDBACK